course/src/main/res/menu/bottom_course_container_menu.xml

```diff
@@ -19,6 +19,12 @@
         android:enabled="true"
         android:icon="@drawable/ic_course_navigation_discussions"/>
 
+    <item
+        android:id="@+id/dates"
+        android:title="@string/course_navigation_dates"
+        android:enabled="true"
+        android:icon="@drawable/ic_course_navigation_discussions"/>
```

Comment:
I think we need to update the icon for this.

Reply:
new icon added.